contracts/liquidity_hub/fee_collector/src/msg.rs

```diff
@@ -41,6 +41,10 @@ pub enum QueryMsg {
     Factories { limit: Option<u32> },
     /// Queries the configuration of this contract
     Config {},
+    Fees {
+        collect_fees_for: QueryFeesFor,
+        accrued: Option<bool>,
```

Comment:
What about renaming `accrued` to `all_time`? That way it is more consistent with the naming we use for the vaults and pools.